Makefile

```diff
@@ -109,12 +110,17 @@ third-party-test-venv: FORCE
 third-party-chpldoc-venv: FORCE
 	cd third-party && $(MAKE) chpldoc-venv
 
+third-party-scspell-venv: FORCE
```

Comment:
I think it's too  late tonight for me to grok `Makefile` vs. `Makefile.devel`, update the PR, and test it.

As I read 8499c9, the idea is that stuff in Makefile.devel won't make it into not-cloned-from-the-github-repo release downloads.  Will util/chplspell end up in the release?  Should it?